x-pack/plugins/apm/server/routes/assistant_functions/route.ts

```diff
@@ -191,7 +191,7 @@ const getApmErrorDocRoute = createApmServerRoute({
   },
 });
 
-interface ApmServicesListItem {
+export interface ApmServicesListItem {
```

Comment:
Is this imported anywhere or why is this needed?

Reply:
I'm not seeing an import anywhere in `8.10` or `main`. Seems it was carried along with the changes from 34c084d

Reply:
It's a deliberate change. I agree it's an add issue and one of those "make this change to satisfy TypeScript" things. You can see the error that I got which prompted me to make this change. If there's a better way to do it I'm all ears:

<img width="977" alt="image" src="https://github.com/elastic/kibana/assets/10602/4724eafe-1479-4c21-a07c-537c9d4e941e">